tsconfig.json

```diff
@@ -19,11 +19,7 @@
     "allowSyntheticDefaultImports": true,
     "allowJs": true,
     "alwaysStrict": true,
-    "paths": {
-      "gatsby-theme-iterative-docs/*": [
-        "./plugins/gatsby-theme-iterative-docs/*"
-      ]
-    }
+    "noEmit": true
```

Comment:
We never emit our typescript so it may as well be in our config.

Also stops the vscode linter from yelling at me for what otherwise would be invalid tsconfig.